react-with-graphql-tutorial.md

```diff
@@ -720,19 +721,19 @@ class App extends Component {
 }
 {{< /highlight >}}
 
-The previous setup for the form, input field(s), a submit button, `onChange()` and `onSubmit()` class methods is a common way to implement forms in React. The only addition is the initial data fetching in the `componentDidMount()` lifecycle method which only helps to improve the user experience by providing an initial state from a backend. In summary, it is a neat boilerplate to [fetch data in React from a third-party API](https://www.robinwieruch.de/react-fetching-data/).
+The previous setup for the form--using input field(s), a submit button, `onChange()` and `onSubmit()` class methods--is a common way to implement forms in React. The only addition is the initial data fetching in the `componentDidMount()` lifecycle method to improve user experience by providing an initial state from the backend. It is a useful foundation for [fetching data from a third-party API in React](https://www.robinwieruch.de/react-fetching-data/).
 
-When you start the application on the command line, you should see the initial state for the `path` in the input field. Furthermore, you should be able to change the state by entering something else in the input field. Nothing should happen on `componentDidMount()` and submitting the form.
+When you start the application on the command line, you should see the initial state for the `path` in the input field. You should be able to change the state by entering something else in the input field. Nothing should happen on `componentDidMount()` and submitting the form.
 
-You might wonder why there is only one input field to grab the information about two things: the organization and repository. When opening up a repository on GitHub, you can see that the organization and repository are encoded in the URL. That's why it is a convenient way to show the same URL pattern for the input field. Moreover, you can split the `organization/repository` later on at the `/` to get values for the organization and repository to perform the GraphQL query request.
+You might wonder why there is only one input field to grab the information about the organization and repository. When opening up a repository on GitHub, you can see that the organization and repository are encoded in the URL, so it becomes a convenient way to show the same URL pattern for the input field. You can also split the `organization/repository` later at the `/` to get these values and perform the GraphQL query request.
 
 ### Exercises:
 
-* if you are not familiar with React, read up *The Road to learn React*
+* If you are not familiar with React, read up *The Road to learn React*
```

Comment:
"If you are not familiar with React, read up *The Road to learn React*" to "If you are not familiar with React, you should checkout *The Road to learn React*"